docs/BROWSERSETUP.md

```diff
@@ -0,0 +1,39 @@
+
+## Example structure of the JSON-File   
+![BrowserSetup](/images/browserSetup.png)
+
+### Browser Setup Elements
+The **browserSetup.json** file consists of several different Browser Setup Elements marked blue in the screenshot.  
+Each Element consists of    
+* an arbitrary chosen element name (e.g. in screenshot "firefox-section-without-os") 
+* an optional Browser Setting entry for the platform (os - marked yellow in the screenshot). 
+* an optional Browser Setting entry for the browser to test (browser - marked yellow in the screenshot). 
+* an optional Browser Setting entry for options (options - marked green in the screenshot).      
+* an optional Browser Setting entry for capabilities (capabilities - marked red in the screenshot). 
+
+
+### Browser Settings
+#### Key "os"  
+When a test should run with OS specific settings the key for "os" must be set, for the specific values please refer to the os specific values on the Selenium web site under [DesiredCapabilities].(https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities)  
+When a test should run without OS specific settings means, the specified "Browser Settings" under this "Browser Setup Element" is valid for all OS (currently Windows, Linux, MAC). It is also possible to mix both options for os-specific and not os-specific. An Error will occur when in both options duplicate entries for the same key exists during a test on a given operating system. In the screenshot we have an example for setting the SSL proxy URL and the SSL proxy port for all OS (see screenshot BrowserSetup-Element "firefox-section-without-os")  
+![BrowserSetup-Element](/images/browserSetupElements.png)  
+and for the Linux OS the proxy type varies (1 instead of 5 see screenshot proxyType) in contrast to Windows OS proxy type.  
+![proxyType](/images/proxyType.png) 
+  
+#### Key "browser"
+When a test should run with browser specific settings the key for "browser" must be set,  
+for the specific browser names, please refer to the Selenium web site under [DesiredCapabilities](https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities).   
+When no browser entry is specified, that means all options and capabilities are defined for all browser vendors.    
+
+#### Key "options"
+When a test should run with browser specific option settings the key for "options" must be set.  
+For the browser specific values please refer to the particular browser vendor.
+
+#### Key "capabilities"
+When a test should run with browser specific capability settings the key for "capabilities" must be set.  
+see screenshot capabilities.  
+![capabilities](/images/capabilities.png).  
+Please refer for the os specific values to the Selenium web site under [DesiredCapabilities](https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities). 
```

Comment:
_For the OS-specific values, please refer to the Selenium web site under [DesiredCapabilities](https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities)._

This seems a bit misleading, because there does not seem to be a section on "OS-specific values". The key "ignoreProtectedModeSettings" seen in the screenshot, for example, is listed under "IE Specific" (I see they don't hyphenate these phrases, either, but I still think they should be :stuck_out_tongue_winking_eye: )